CenterForOpenScience/ember-osf-web
# [ENG-2546] Add route for My Registrations page

- Ticket: [ENG-2546](https://openscience.atlassian.net/browse/ENG-2546)
- Feature flag: n/a

## Purpose

We will want to develop on the My Registrations page, and for that we need a route and template. This makes those.

<img width="1410" alt="Screen Shot 2021-02-01 at 2 48 33 PM" src="https://user-images.githubusercontent.com/6599111/106510216-a45a7700-649c-11eb-9975-236163c3fde8.png">


## Summary of Changes

1. Add the route
2. Add the route.ts and have it gather analytics for people accessing the page
3. Add a blank template

## Side Effects

Nope

## QA Notes

This is not, in its current form, very QA-able. But you could go to `/registries/my-registrations` to see a mostly blank page if you really wanted.
